main.py

```diff
@@ -2,6 +2,44 @@
 
 from django.db.models import QuerySet
 
+from db.models import Genre, Actor
+
 
 def main() -> QuerySet:
-    pass
+    def fill_in_tables() -> None:
+        genres = ["Western", "Action", "Dramma"]
+
+        for genre in genres:
+            Genre.objects.create(name=genre)
+
+        actors = [
+            ("George", "Klooney"),
+            ("Kianu", "Reaves"),
+            ("Scarlett", "Keegan"),
+            ("Will", "Smith"),
+            ("Jaden", "Smith"),
+            ("Scarlett", "Johansson")
+        ]
+
+        for actor in actors:
+            Actor.objects.create(first_name=actor[0], last_name=actor[1])
```

Comment:
use unpacking here for better readability. Some hints are here https://github.com/mate-academy/py-genres-and-actors/blob/master/checklist.md